rest-service/manager_rest/security/authentication.py

```diff
@@ -77,7 +77,8 @@ def authenticate(self, request):
             # (User + Password), otherwise the counter will be reset on
             # every UI refresh (every 4 sec) and accounts won't be locked.
             user.failed_logins_counter = 0
-        user.last_login_at = datetime.now()
+        if not readonly:
+            user.last_login_at = datetime.now()
         user_datastore.commit()
```

Comment:
so this should not commit at all if readonly, because it'll just break if anything is changed on the user

however now we note that the other thing that can be changed here is the failed logins counter, and so we realize that this has security implications because a readonly login allows you to circumvent the failed login block...